projects/wave-core/src/lib/operators/dialogs/data-repository/mapping-source.model.ts

```diff
@@ -154,6 +165,8 @@ export interface MappingSourceVectorLayerDict {
         license: string,
         citation: string,
     };
+    time_start?: Time;
+    time_end?: Time;
```

Comment:
Same here.

Reply:
fixed in 9919298afbd39ba8285c6946f1eec93e1c65b5ba